### PR TITLE
learn troubleshoot

### PR DESCRIPTION
**Pull request by SijoGeorge7012**
image
image
image

### CODE REVIEW

File: 001troubleshooting.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    <img src="image//image24.png" alt=""> <br>
+                    <label for="" id="label2">2. Click on the Raw button, then You'll get the URL for your HTML file.</label>
+                </section>
+                <section>
+                    <p id="label2">* Check By File Upload:- You can upload a file from your Local Drive and check</p>
+                    <img src="image/image26.png" alt="">
+                </section>
+                <section>
+                    <p id="label2">* Check By Text Input:- You can copy paste your codes as text and check.</p>
+                    <img src="image/image27.png" alt="">
```
Comment (wandercode) [inline 001troubleshooting.html:46]: include the check by text input part on previous slide. 
only tasks should reamin in this slide.

Comment (wandercode) [inline 001troubleshooting.html:50]: include the check by text input part on previous slide. 
only tasks should remain in this slide.

Comment (wandercode) [inline 001troubleshooting.html:50]: include the check by text input part on previous slide. 
only tasks should remain in this slide.

**Review by wandercode — CHANGES_REQUESTED** [review]

File: 001troubleshooting.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	</head>
+	<body>
+		<div class="reveal">
+			<div class="slides">
+				<section>
+                    <h3>Validate your HTML code</h3>
+                </section>
+                <section>
+                    <section>
+                    <p id="para">Please Clone the <a href="https://github.com/brainchildservices/Troubleshooting" target="_blank">Troubleshooting Reposittory</a>  to your Local Storage, to Continue
```
Comment (wandercode) [inline 001troubleshooting.html]: please make the link to open on another tab

**Review by wandercode — CHANGES_REQUESTED** [review]

**Review by VishnuKulakkattu — APPROVED** [review]

File: 001troubleshooting.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                <section>
+                    <p id="para">Go to <a href="https://validator.w3.org/nu/" target="_blank">HTML Validator</a>  and upload your Edited file and check to find your code is in proper way.</p>
+                    <p id="para">For Validating a CSS, you Have 3 Options</p>
+                    <ul id="li1">
+                        <li>Check By Address :- You can Use a URL of Your File. or you can use the Github Link, like this</li>
+                    </ul>
+                    <label for="" id="label2"> 1. Once you open your .html file in your github Repository, you can see a button named Raw like this,</label>
+                    <img src="image//image24.png" alt=""> <br>
+                    <label for="" id="label2">2. Click on the Raw button, then You'll get the URL for your HTML file.</label>
+                    <p id="label2">* Check By File Upload:- You can upload a file from your Local Drive and check</p>
```
Comment (VishnuKulakkattu) [inline 001troubleshooting.html]: Kindly arrange the Points & Screenshot in a single slide

**Review by VishnuKulakkattu — CHANGES_REQUESTED** [review]

**Pull request merged**